pytest_plugins/factory_collection.py

```diff
@@ -0,0 +1,41 @@
+import re
+from inspect import getmembers
+from inspect import isfunction
+
+from robottelo.config import settings
+
+
+def pytest_configure(config):
+    """Register markers related to testimony tokens"""
+    marker = 'factory_instance: Test uses a fresh satellite or Capsule instance deployed by broker'
+    config.addinivalue_line("markers", marker)
+
+
+def pytest_collection_modifyitems(session, items, config):
+    from pytest_fixtures.core import factory
+
+    deselected = []
```

Comment:
why track what's been deselected?

Reply:
So that it shows in the pytest collection as to how many are deselected.  This is the standard we are following in all our collection plugins, so here as well !